edx_event_bus_kafka/internal/producer.py

```diff
@@ -311,7 +316,8 @@ def poll_indefinitely(api_weakref: KafkaEventProducer):
         try:
             api_object.producer.poll(0)
         except BaseException:
-            logger.warning("Event bus producer polling loop encountered exception (continuing)", exc_info=True)
+            logger.exception("Event bus producer polling loop encountered exception (continuing)", exc_info=True)
```

Comment:
I think we don't need exc_info when using the exception method
```suggestion
            logger.exception("Event bus producer polling loop encountered exception (continuing)")
```